components/navigation/navigation.css

```diff
@@ -0,0 +1,109 @@
+.header {
+  width: 100%;
+  position: sticky;
+  background-color: var(--primary-light-color);
+  z-index: 9999;
+  position: sticky;
+  top: 0;
+  left: 0;
+  padding: var(--size-9) 0;
+}
+
+.nav-main {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 var(--size-9) ;
+  margin: 0 auto;
+}
+
+.logo {
+  font-weight: var(--weight-bold);
+  font-size: var(--size-13);
+  color: white;
+}
+
+.hamburger .fas {
+  color: var(--color-white);
+  font-size: var(--size-13);
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.nav-list {
+  color: var(--color-white);
+  flex-basis: 30%;
+  margin-left: auto;
+}
+
+.ecom-nav-cta{
+    outline: var(--size-2) solid var(--color-white);
+
+}
+
+.nav-item {
+  display: inline-block;
+}
+
+.nav-item:nth-child(n + 2) {
+  margin: 0 var(--size-11);
+}
+
+.nav-item:first-child {
+  margin-right: auto;
+}
+
+.search-input {
+  margin: 0 auto;
+  display: flex;
+  flex-basis: 45%;
+  position: relative;
+}
+
+.search-icon {
+  font-size: var(--size-11);
+  margin: auto 0;
+  position: absolute;
+  top: 25%;
+  left: 3%;
```

Comment:
doesn't it look like being too specific?

Reply:
Correct!
Have recitified this with proper positioning, thank you for pointing it out.